### PR TITLE
move to connect openAPI client

### PR DESCRIPTION
**Pull request by Sainomori**

### CODE REVIEW

File: backend/connect_client.py [hunk 1]
```diff
@@ -1,123 +1,87 @@
...
+        Fetches the first page from the "/public/v1/devops/pages" endpoint of the API.
+        If the pages array is empty, it logs an error and raises an exception.
+
+        Returns:
+            Dict[str, Any]: A dictionary represents a page.
+
+        Raises:
+            Exception: If the pages array is empty, an exception is raised with a relevant error message.
+        """
+        page = await self.connect_client("devops").collection("pages").all().first()
```
Comment (ffaraone) [inline backend/connect_client.py]: I would suggest to filter by `integration_point = 'customer'`. For the demo we will have just 1 customer page plus other pages.

File: backend/connect_client.py [hunk 2]
```diff
@@ -1,123 +1,87 @@
...
+        Raises:
+            Exception: If the "code" is not found in the response JSON,
+            an exception is raised with a relevant error message.
+        """
+        ext_id = re.search(r"EXT-\d+-\d+", login_url)
+        if not ext_id:
+            raise Exception("EXT-id not found in login_url")
+
+        response_json = (
+            await self.connect_client("auth").eaas[ext_id.group()]("login").post()
```
Comment (ffaraone) [inline backend/connect_client.py]: you can also rewrite this the following way:
```
from connect.client.models import AsyncAction

login_action = AsyncAction(self.connect_client, login_url)
response_json = await login_action.post()
```

**Review by ffaraone — CHANGES_REQUESTED** [review]

File: backend/connect_client.py (hunk 1 above)
Comment (Sainomori) [reply backend/connect_client.py]: done

File: backend/connect_client.py (hunk 2 above)
Comment (Sainomori) [reply backend/connect_client.py]: fixed with moving to ext-id instead of login url

**Review by ffaraone — APPROVED** [review]

**Pull request merged**